README.md

```diff
@@ -39,7 +39,21 @@ This is all possible inside `todo.txt`.
 
 ## `todo.txt` format rules
 
-<img src="./description.svg" width="100%" height="500">
+```
+┌─────────────────── Optional - Marks completion 
+│  ┌──────────────── Optional - Marks priority 
+│  │      ┌─────────────── Optional - Completion date 
+│  │      │          ┌──── Optional - Creation date
+│  │      │          │     (must be specified if completion date is)
+│  │      │          │             ┌─── Mandatory - Description
+│  │      │          │             │    Tags (optional) can be placed anywhere within it.
+│  │  ╭───┴───╮  ╭───┴───╮  ╭──────┴───────────────────────────────────────────────╮
+x (A) 2016-05-20 2016-04-30 measure space for +chapelShelving @chapel due:2016-05-30
```

Comment:
Three additional (minor) suggestions:

- Move the line pointing up from the "Description" field a bit to the left, so the explanatory text doesn't extend past the diagram;
- Use angled corners instead of rounded corners, to match the rest of the diagram
- Expand the first two ranges so they cover the entire dates (currently they don't include the final digit, for some reason) 

```suggestion
│  │      │          │        ┌─── Mandatory - Description
│  │      │          │        │    Tags (optional) can be placed anywhere within it.
│  │  ┌───┴────┐ ┌───┴────┐ ┌─┴────────────────────────────────────────────────────┐
x (A) 2016-05-20 2016-04-30 measure space for +chapelShelving @chapel due:2016-05-30
```

Finally, I'd recommend removing the end-of-line whitespace on lines 43, 44, 45 and 52.

Reply:
Sensible changes